src/TLuaInterpreter.cpp

```diff
@@ -12351,21 +12366,16 @@ std::pair<bool, bool> TLuaInterpreter::callMultiReturnBool(const QString& functi
 }
 
 // No documentation available in wiki - internal function
-bool TLuaInterpreter::callCmdLineAction(const int func, QString text)
+bool TLuaInterpreter::callReference(lua_State* L, QString name, int parameters)
 {
-    lua_State* L = pGlobalLua;
-    lua_rawgeti(L, LUA_REGISTRYINDEX, func);
     int error = 0;
-    lua_pushstring(L, text.toUtf8().constData());
-    error = lua_pcall(L, 1, LUA_MULTRET, 0);
+    error = lua_pcall(L, parameters, LUA_MULTRET, 0);
     if (error) {
         std::string err = "";
         if (lua_isstring(L, -1)) {
             err += lua_tostring(L, -1);
         }
-
-        QString function = "setCmdLineAction";
-        QString name = "cmd line Action";
+        QString function = "anonymous Lua function";
         logError(err, name, function);
         if (mudlet::debugMode) {
             TDebug(QColor(Qt::white), QColor(Qt::red)) << "LUA: ERROR running script " << function << " (" << function << ")\nError: " << err.c_str() << "\n" >> 0;
```

Comment:
Since you have change the function name from `setCmdLineAction` to `anonymous Lua function` it means that the Central Debug Console output will be different - is this intentional?

Also beware that PR #4781 will be clobbering **every** `TDebug(QColor, QColor)` instance throughout the application...

Reply:
Yes, this was intentional as I think it was wrong before. setCmdLineAction functions are "anonymous lua functions" the same as for example tempTimer functions.

I don't think it won't create any issue with #4781 as it will only log to the central debug console if there is an error ( this behaviour is the same as before)

Reply:
@SlySven That's that you get from mixing a small unproblematic API change together with a debatable enhancement in the same PR.. ;)